local_settings.py.example

```diff
@@ -105,6 +105,56 @@ RELATE_EDITABLE_INST_ID_BEFORE_VERIFICATION = True
 
 # }}}
 
+
+# {{{ user full_name format
+
+# RELATE's default fullname format is "'%s %s' % (first_name, last_name)",
```

Comment:
I just copied it from [here](https://github.com/inducer/relate/blob/master/accounts/models.py#L142)